README.md

```diff
@@ -0,0 +1,175 @@
+# Management user account and transaction inside laravel app
+
+[![Latest Version on Packagist][ico-version]][link-packagist]
+[![Total Downloads][ico-downloads]][link-downloads]
+[![Software License][ico-license]][link-license]
+[![Testing status][ico-workflow-test]][link-workflow-test]
+[![Open API][ico-open-api]][link-open-api]
+
+## Introduction
+
+The  dnj/laravel-account Public package provides easy way to manage  account and transaction of the users of your app. The Package stores all data in the accounts and transactions table.
+* Latest versions of PHP and PHPUnit and PHPCsFixer
+* Best practices applied:
+    * [`README.md`][link-readme] (badges included)
+    * [`LICENSE`][link-license]
+    * [`composer.json`][link-composer-json]
+    * [`phpunit.xml`][link-phpunit]
+    * [`.gitignore`][link-gitignore]
+    * [`.php-cs-fixer.php`][link-phpcsfixer]
+    * [`openAPI`][link-phpcsfixer]
+* Some useful resources to start coding
+
+
+## Here's a demo of how you can use it:
+```php
+$account = new AccountManager();
+$account->create(
+    'this is a firest account',
+    $currency_id,
+    Auth::user()->id,
+    AccountStatus::ACTIVE,
+    false,
+    true,
+    ["key" => "value"]
+);
+```
+## Installation
+You can install the package via composer:
+```bash
+composer require dnj/laravel-account
+```
+
+The package will automatically register itself.
+
+
+After this you can create the `accounts and transactions` table by running the migrations:
+
+```bash
+php artisan migrate
+```
+
+You can optionally publish the config file with:
+
+```bash
+php artisan vendor:publish --provider="dnj\laravel-account\AccountServiceProvider" --tag="config"
+```
+
+### Account usage:
```

Comment:
These code snippets are not good enough.
Put yourself instead of a newcomer and see these going to be useful or not.